test/init.sh

```diff
@@ -27,7 +27,7 @@ oneTimeSetUp() {
 }
 
 oneTimeTearDown() {
-  printf "\nstoping tendermint..."
+  printf "\nstoping..."
```

Comment:
I was trying to indicate that "stopping" is spelled wrong, not that we shouldn't say "tendermint" :)